addons/velaux/resources/server.cue

```diff
@@ -23,7 +23,7 @@ _nginxTrait: *[
 				http: {
 					"/": 8000
 				}
-				class: "nginx"
+				class: parameter["ingressClass"]
```

Comment:
maybe you can reuse the gatewayDriver field.

Reply:
Noted, updated to use gatewayDriver and works on my cluser. The e2e test however is timing out, seeing why

Reply:
Tests are failing because the version of the addon is also used for the image tag, do you prefer creating a new tag v1.9.4 or parameterising the image tag and allow for addons upgrades independently from velaux

Reply:
@charlie0129 what do you think. I get is nice for the tag/version of addon to map but in reality there can be addon changes with not image release